tests/integration/targets/dcnm_network/tests/dcnm/merged.yaml

```diff
@@ -645,6 +645,121 @@
     fabric: "{{ ansible_it_fabric }}"
     state: deleted
 
+- name: MERGED - Create L3 networks along with all dhcp, arp options without attach
+  cisco.dcnm.dcnm_network:
```

Comment:
Looks like this is a new test case. Did you check if the existing test cases for merge work properly?

Reply:
Yes all the exiting cases are fine. This specific scenario where we create network first and attach switched subsequently in a different operation was not cover before for merge, so added a case for that.